pythonequipmentdrivers/resource_collections.py

```diff
@@ -1,9 +1,15 @@
-from pathlib import Path
 import json
-from pyvisa import VisaIOError
-from . import errors
 from importlib import import_module
-from typing import Union, Tuple
+from pathlib import Path
+from types import SimpleNamespace
+from typing import Dict, Tuple, Union, Iterator
+
+
+from pyvisa import VisaIOError
```

Comment:
Remove dependency on pyvisa from this file, should be encapsulated by the new VisaResource base class